.github/integ-config/utils.yml

```diff
@@ -22,3 +22,20 @@ datastore_auth_scenarios_sample_spec:
   - public-auth-iam public-auth-iam
   - owner-custom-claim-default owner-custom-claim-default
   - owner-custom-field-default owner-custom-field-default
+
+datastore_auth_scenarios_sample_spec_v2:
+  - v2/owner-based-default-v2 owner-based-default # TODO: Add `owner-based-operations` when tests pass
+  - v2/static-user-pool-groups-default-v2 static-user-pool-groups-default
+  - v2/static-user-pool-groups-operations-v2 static-user-pool-groups-operations
+  - v2/owner-and-group-different-models-default-v2 owner-and-group-different-models-default
+  - v2/owner-and-group-same-model-default-v2 owner-and-group-same-model-default
+  - v2/owner-and-group-same-model-operations-v2 owner-and-group-same-model-operations
+  - v2/private-auth-iam-v2 private-auth-iam
+  - v2/public-auth-default-v2 public-auth-default
+  - v2/public-auth-iam-v2 public-auth-iam
+  - v2/owner-custom-claim-default-v2 owner-custom-claim-default
+  - v2/owner-custom-field-default-v2 owner-custom-field-default
+  - v2/dynamic-user-pool-groups-default-v2 dynamic-user-pool-groups-default
+  - v2/private-auth-default-v2 private-auth-default
+  # TODO: The below test fails and ends up failing all the other tests in this group
+  # - v2/dynamic-user-pool-groups-owner-based-default-v2 dynamic-user-pool-groups-owner-based-default
```

Comment:
Running this test seems to crash gh actions and stops all others jobs from running as well. Will need to investigate further. Commenting for now
[Please see sample run for reference](https://github.com/stocaaro/amplify-js/actions/runs/5472416024)